src/transformers/lib_integrations/peft/peft_mixin.py

```diff
@@ -113,7 +113,7 @@ def load_adapter(
             offload_index (`int`, `optional`):
                 `offload_index` argument to be passed to `accelerate.dispatch_model` method.
         """
-        check_peft_version(min_version="0.4.0")
+        check_peft_version(min_version="0.5.0")
```

Comment:
Maybe set this on top of this file? I don't knnow if this is going to be changed along time, but if so, a single place of definition is easier for life.

Reply:
the problem is that we always import the `PeftMixin` object, meaning that check would be always called and it will raise an error if PEFT is not installed I think

Reply:
Discussed offline and got it, will push somethning